prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+storage.
+
+Each data storage drive will be managed, but regardless of the partitioning
+scheme, all data drive partitions will be mounted at the ``/srv/`` path.
+The **data storage drive must be a different disk drive than the system disk drive**
+where the operating system is installed. **One partition of each data
+drive can handle one or many shared resources.**
+
+==============  ==========  ===========  ========================================
+ Partition       Min size    Best size    Mandatory
+==============  ==========  ===========  ========================================
+ ``/srv/<*>``     100Mib       bigger     Yes, size as need! example 128Gib
+==============  ==========  ===========  ========================================
+
+Technical notes
+---------------
+
+Take into **consideration that as more exquisite the customization as less supported
+could be.**.
+
+|omv| allows for minimal deployment and use of the system with very minimal
+hardware/software requirements, at the cost of the key knowledge, but other NAS
+systems allow ease of use at the cost of high requirements. |omv| allows both as
+well in a well balanced use cases! [7]_
+
+About minimal or custom setups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It can be noted that |omv| system can be installed on just 4Gib of system partition,
+with no more than 1Gib of RAM and using a WiFi or USB connection to access it
+remotely (cos is a NAS system of course), on any small device such as Raspberry
+ones which are ARM, or old i386s machines no matter is those are 32 or 64 bits, 
+but of course as more exquisite the configuration more knowledge will be need
+as per `issue comment #131 <https://github.com/openmediavault/openmediavault-docs/issues/131#issuecomment-2546765841>`_.
+
+Technically OMV can be installed on a single storage disk, this is possible if
+it is done on a previously configured Debian system, with a free partition apart
+from the 3 necessary system partitions, previously formatted and configured.
+Of course this is only possible for skilled linux users.
+
+Drive Storage technical details
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+System Drives are not managed as same of Data Drives. System drives are not so
+intensively used, but Data Drives will need triks to extend the useful life.
+
+If you use a Flash Drive, select one with static wear leveling 6, without this
+the drive will have a very short lifetime. It is also recommended to install and
+activate the Flash Memory plugin.
+
+In the same Solid State Disk, or rather Drive (SSD) for x86 architectures, is
+usually only recognized properly by the BIOS or UEFI, when in the BIOS/UEFI the
+feature AHCI has been activated for SATA (instead of IDE). Modern computers has
+as default, so that's fine. But on old machines the default might/could be IDE.
+About ARM based computers this are not a problem, when used SATA interfaces, but
+EMMC ones could need tune up cos are threated as Flash drives.
```

Comment:
```suggestion
eMMC ones could need tune up because they are threated as Flash drives.
```

Is `threated` the correct word here?

Reply:
> Is `threated` the correct word here?

should be `treated`

Reply:
No. i search due your suggestion and seems ok: 

https://www.oed.com/search/dictionary/?scope=Entries&q=treated

![image](https://github.com/user-attachments/assets/e38f39e2-e114-4539-b0a3-5aba34d4ef8c)